cmd/gitops/root/cmd.go

```diff
@@ -85,6 +88,11 @@ func RootCmd(client *resty.Client) *cobra.Command {
 `, wego.DefaultNamespace),
 		PersistentPreRun: func(cmd *cobra.Command, args []string) {
 			configureLogger()
+			err := initViper(cmd)
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "Error initing viper: %v\n", err)
+				os.Exit(1)
+			}
```

Comment:
You could use [log.Fatalf()](https://pkg.go.dev/log#Fatalf) here, it writes to Stderr, and exits with code 1.

Reply:
Nice! I copied the existing pattern in the file to make minimal changes, will test this out too!